sdk/communication/azure-communication-identity/README.md

```diff
@@ -127,6 +116,19 @@ System.out.println("User token value: " + userToken.getToken());
 System.out.println("Expires at: " + userToken.getExpiresAt());
 ```
 
+### Create a new user and token in a single request
+For convenience, use `createUserAndToken` to create a new user and issue a token with one function call. This translates into a single web request as opposed to creating a user first and then issuing a token.
+<!-- embedme ./src/samples/java/com/azure/communication/identity/ReadmeSamples.java#L101-L107 -->
+```java
+// Define a list of communication token scopes
+List<CommunicationTokenScope> scopes =
+    new ArrayList<>(Arrays.asList(CommunicationTokenScope.CHAT));
```

Comment:
My java is basic, but now that we accept Iterator, do we still need to create ArrayList?

Reply:
You're absolutely right! We do not need a new ArrayList :) thank you